.env.development

```diff
@@ -2,3 +2,9 @@
 # https://create-react-app.dev/docs/adding-custom-environment-variables/
 
 REACT_APP_GATEWAY=http://localhost:8080/api/v1/
+
+# To login using grip:
+# REACT_APP_AUTH_URL="https://auth.grip-on-it.com/v2/rjsfm52t/oidc/idp/authorize?response_type=code&scope=openid&client_id=d3d664c7-bb33-4bf0-b7c9-b8bdf1199b76&redirect_uri=http://localhost:2999/"
+REACT_APP_AUTH_URL=/login/callback/?code=1234567890abcdefghijkl
```

Comment:
Think `?code=1234567890abcdefghijkl` is added by Grip

Reply:
You can use this URL (a direct link to the callback), to bypass GRIP.